src/Route/AuthenticationRoute.php

```diff
@@ -53,7 +53,7 @@ class AuthenticationRoute extends AbstractRoute
      * @throws ResourceNotFoundException
      * @throws UnauthorizedException
      */
-    public function login(string $apiKey, string $userPIN = null): string
+    public function login(string $apiKey, string ?$userPIN = null): string
```

Comment:
This change is breaking tests as the ? is in the wrong place :)

Reply:
Yes, I'll schedule some time in the coming days to update the tests and remove unsupported php versions from the test runs.

The library works so stable for me personally that I tend to forget it exists...